badger_utils/coingecko_utils.py

```diff
@@ -9,16 +9,23 @@
 from badger_utils.registry import registry
 
 
+TOKEN_TO_ID_MAP = {
+    web3.toChecksumAddress(registry.tokens.wbtc): "wrapped-bitcoin",
```

Comment:
I think we should add the following assets as well as we currently use them in our infra or may use them in the future:
- RenBTC
- sBTC
- tBTC
- bBTC
- oBTC
- pBTC
- hBTC
- MTA
- BNB
- MATIC
- AVAX
- AAVE
- COMP
- OHM
- SWAPR

Reply:
Done